src/aiida/tools/pytest_fixtures/orm.py

```diff
@@ -13,48 +13,55 @@
 
 @pytest.fixture(scope='session')
 def ssh_key(tmp_path_factory) -> t.Generator[pathlib.Path, None, None]:
-    """Generate a temporary SSH key pair for the test session and return the filepath of the private key.
+    """Returns a SSH key for the test session. If the environment variable ``AIIDA_PYTEST_SSH_KEY`` is set we take the
+    key from this path otherwise we generate a temporary SSH key pair for the test session and return the filepath of
+    the private key.
```

Comment:
```suggestion
    """Returns a SSH key for the test session.
    
    If the environment variable ``AIIDA_PYTEST_SSH_KEY`` is set we take the
    key from this path otherwise we generate a temporary SSH key pair for the test session and return the filepath of
    the private key.
```